postgresql@14.rb

```diff
@@ -23,10 +23,10 @@ class PostgresqlAT14 < Formula
   depends_on "pkg-config" => :build
 
   depends_on "gettext"
-  depends_on "icu4c"
+  depends_on "icu4c@76"
```

Comment:
I also saw this dependency was changed to this specific version in that other formula - so I included it for now, but not sure about whether it's important.